test/next/src/app/layout.tsx

```diff
@@ -2,8 +2,6 @@ import { Inter } from 'next/font/google';
 
 const inter = Inter({ subsets: ['latin'] });
 
-export const dynamic = 'force-dynamic';
```

Comment:
@tcc-sejohnson I had to remove this in order to reproduce issue-119. I created and committed a reproduction for 119. I was only able to reproduce by not having force-dynamic declared globally.

I instead added it to the selected pages manually.